### PR TITLE
Azure theme bump

### PR DESCRIPTION
**Pull request by micahgodbolt**
Bumping Azure theme to 6.0 so that next release will be 6.0.1. 

This will let us look at creating a 7.0 release in the future that is the delta between our fluent 7.0 designs and the target Azure styles (which are moving more fluent). 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9153)

### CODE REVIEW

**Comment by chieftn** [comment]
Looks good from our side.  Thank you!

**Comment by micahgodbolt** [comment]
Why is this not building :(

**Comment by micahgodbolt** [comment]
missed some change files. got those in now.

File: scripts/typescript-compatibility-build.yml [hunk 1]
```diff
@@ -171,21 +171,21 @@ jobs:
...
-            } else {
-              if($i -match 1) {
-                $compatibleTsVersions += $line;
-              } else {
-                $compatibleTsVersions += ", ";
-                $compatibleTsVersions += $line;
-              }
-            }
-            $i++;
+          if($i -match 0) {
```
Comment (cliffkoh) [inline scripts/typescript-compatibility-build.yml]: why change this file? can you revert?

Comment (micahgodbolt) [reply scripts/typescript-compatibility-build.yml]: seems to have tweaked the formatting for some reason

**Pull request merged**